src/lucky/router.cr

```diff
@@ -1,19 +1,32 @@
 class Lucky::Router
-  INSTANCE = new
+  alias RouteMatch = LuckyRouter::Match(Lucky::Action.class) | Lucky::FallbackRoute | Nil
+  private class_property instance = new
 
   getter :routes
+  getter fallback : Lucky::FallbackRoute?
```

Comment:
Also I think we should pass in the Fallback **class** if possible to match what we do with regular routes